src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,19 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: 1024px;
```

Comment:
Even though this is not said explicitly in checklist 11, I think it is meant that the width of the parent container (.cards) should be calculated dynamically depending on the width of its elements. But this is just my reasoning ofc. I may be wrong here.

Reply:
Hi @AndreiSuc and @59MrRobot I think from the checklist said the there should be 4 columns. I think we should keep it fixed. Eish I tried to use flex-basis: 25% to have 4 columns but it goes out of hand when the viewport is resized. I don't know maybe there is a way.